demo/package.json

```diff
@@ -5,8 +5,9 @@
   "type": "module",
   "scripts": {
     "dev": "VITE_DEMO_SRC='./lite/index.html' VITE_TITLE='Lite' vite",
-    "build": "tsc && vite build --base=./",
+    "build": "tsc && VITE_DEMO_SRC='http://localhost:8888' VITE_TITLE='Local' vite build --base=./",
     "build:ghpages": "tsc && VITE_DEMO_SRC='./lite/index.html' VITE_TITLE='Lite' vite build --base=./",
+    "build:lite": "jupyter lite build --contents ../README.md --contents ./example.ipynb --output-dir ./public/lite",
```

Comment:
Should CI also use this script for building the JupyterLite archive for GitHub Pages here?

https://github.com/TileDB-Inc/jupyter-iframe-commands/blob/f8596a03f8740e0a143c94ee07c88b95de511add/.github/workflows/build.yml#L184

If so, maybe we could also move `overrides.json` and `jupyter-lite.json` to the demo, so all demo files are in one place.

Reply:
Maybe we can do that in a separate PR though.